test/integration/comments.test.js

```diff
@@ -29,10 +29,18 @@ let profileUrl = `${rootUrl}/users/${username2}`;
 let studioId = process.env.COMMENT_STUDIO_ID || 10005646;
 let studioUrl = `${rootUrl}/studios/${studioId}/comments`;
 
+// setup comments to leave
 let date = new Date();
 let dateString = `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()} ` +
 `${date.getHours()}:${date.getMinutes()}:${date.getSeconds()}`;
 let buildNumber = process.env.CIRCLE_BUILD_NUM || dateString;
+let projectComment = buildNumber + ' project';
+let profileComment = buildNumber + ' profile';
+let studioComment = buildNumber + ' studio';
+
+// let projectReply = buildNumber + ' project reply';
```

Comment:
Is there a reason to keep these commented out lines?

Reply:
Oh, they are for upcoming tests.  I just didn't get that far yet.  I could take them out for this PR, though.